scripts/split_repo_migration/delete-everything-split-repo.sh

```diff
@@ -0,0 +1,145 @@
+
+git clone "git@github.com:googleapis/${SPLIT_REPO}.git" $SPLIT_REPO_DIR
+
+cd $SPLIT_REPO_DIR
+
+# Create a git branch
+git checkout -b 'migrate-library'
+
+# Quick check to make sure there are no handwritten samples. 
+# Handwritten samples should be migrated to python-docs-samples before repository code is deleted.
+# Exclude the samples/generated_samples/* directory which contains autogenerated samples.
+NUM_PY_SAMPLES_FILES=$(find samples -type f -name "*.py" ! -path "samples/generated_samples/*" | grep -v "nox.*\.py" | grep -v "__init__\.py" | wc -l)
+
+# Fail if there are handwritten samples.
+if [[ $NUM_PY_SAMPLES_FILES -ne "0" ]]; then
+  echo "Please migrate handwritten samples to python-docs-samples before deleting the source."
+  exit 1
+fi
+
+# Fail if this repository has a customized version of the autogenerated code.
+# This specific check is to ensure that autogenerated files are not excluded.
+STANDARD_EXCLUDES=$(grep owlbot.py -e 'excludes=\["\*\*/gapic_version.py"]' | wc -l)
+
+if [[ $STANDARD_EXCLUDES != "1" ]]; then
+  echo "A non-standard value for \`excludes\` is not supported in the repository"
+  exit 1
+fi
+
+# Fail if this repository has a customized version of the autogenerated code.
+# This specific check is to make sure that the owlbot.py file does not contain `s.replace` or `sed`
+# which has historically been used to modify the generated code.
+NUM_STRING_REPLACEMENTS=$(grep owlbot.py -e 's.replace' | wc -l)
+if [[ $NUM_STRING_REPLACEMENTS != "0" ]]; then
+  echo "s.replace() is not supported in the python mono repository"
+  exit 1
+fi
+
+NUM_STRING_REPLACEMENTS=$(grep owlbot.py -we 'sed' | wc -l)
+if [[ $NUM_STRING_REPLACEMENTS != "0" ]]; then
+  echo "s.replace() is not supported in the python mono repository"
+  exit 1
+fi
+
+# Delete everything except LICENCE, SECURITY.md, README.rst, .git and .repo-metadata.json
+ls -A | grep -v 'LICENCE\|SECURITY.md\|README.rst\|.git\|.repo-metadata.json' | xargs rm -rf
+
+# Additional cleanup for files/folders with .git prefix
+rm -rf .github
+rm -rf .gitignore
+
+# Append a prefix to the README to state that this repository has been archived and the source has moved.
+README_RST="${SPLIT_REPO_DIR}/README.rst"
+
+if [[ ! -f $README_RST ]]; then
+  echo "README.rst file not found"
+  exit 1
+fi
+
+cat > ${SPLIT_REPO_DIR}/README_CONTENT_PREFIX.rst << EOF
+:**NOTE**: **This github repository is archived. The repository contents and history have moved to** \`google-cloud-python\`_.
+
+.. _google-cloud-python: https://github.com/googleapis/google-cloud-python/tree/main/packages/${ARTIFACT_NAME}
+
+
+EOF
+
+# Create temp file for the prefix that we want to add to the README content.
+README_CONTENT_PREFIX_RST="${SPLIT_REPO_DIR}/README_CONTENT_PREFIX.rst"
+
+# Concatenate the README prefix and the README content into the existing README
+cat $README_CONTENT_PREFIX_RST $README_RST | sponge $README_RST
+
+# Remove temp file
+rm -rf $README_CONTENT_PREFIX_RST
+
```

Comment:
```suggestion
{ cat - $README_RST | sponge $README_RST ; } << EOF
:**NOTE**: **This github repository is archived. The repository contents and history have moved to** \`google-cloud-python\`_.

.. _google-cloud-python: https://github.com/googleapis/google-cloud-python/tree/main/packages/${ARTIFACT_NAME}


EOF

```

This is simpler: it `cat`s first `stdin` and then the contents of the file, piping the output into `sponge`. We enclose the two commands in braces so the `stdin` for the whole group (hence to `cat`) is the heredoc (otherwise `<<` would bind to `sponge`)

Reply:
Done in [2462e2e](https://github.com/googleapis/google-cloud-python/pull/10879/commits/2462e2ef0028d827d542a7ab86c9593bb75376fc)